sandbox/grommet-app/src/pages/layouts/kinds/Insider/Stories/stories/CodeWars.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Image } from 'grommet';
+import image from '../../../../../../assets/codewars.png';
+import { StoryCard } from '../components';
+
+export const CodeWars = ({...rest}) => {
+    return (
+        <StoryCard 
+            level={3}
+            size={undefined}
```

Comment:
Design calls for this section's headings to render as 24px (1.5rem) but these are rendering as 18px (1.125rem)